docs/content/latest/api/ysql/extensions.md

```diff
@@ -23,159 +23,202 @@ These are extensions that are included in the standard YugabyteDB distribution a
+    ```
+     id | content | username |          moddate
+    ----+---------+----------+----------------------------
+      1 | desc1   | yugabyte | 2019-09-13 16:55:53.969907
+      2 | desc2   | postgres | 2019-09-13 16:55:53.983306
+      3 | desc3   | postgres | 2019-09-13 16:55:53.98658
+      4 | desc4   | yugabyte | 2019-09-13 16:55:53.991315
+    (4 rows)
+    ```
 
-{{< /note >}}
+    {{< note title="Note" >}}
 
-3. Update some rows. Should update both `username`  and `moddate` accordingly.
-
-```plpgsql
-UPDATE spi_test SET content = 'desc1_updated' WHERE id = 1;
-UPDATE spi_test SET content = 'desc3_updated' WHERE id = 3;
-SELECT * FROM spi_test ORDER BY id;
- id |    content    | username |          moddate
-----+---------------+----------+----------------------------
-  1 | desc1_updated | yugabyte | 2019-09-13 16:56:27.623513
-  2 | desc2         | postgres | 2019-09-13 16:55:53.983306
-  3 | desc3_updated | yugabyte | 2019-09-13 16:56:27.634099
-  4 | desc4         | yugabyte | 2019-09-13 16:55:53.991315
-(4 rows)
-```
+The `yugabyte` and (for compatibility) `postgres` YSQL users are created by default.
+
+    {{< /note >}}
 
-For more information, see [`spi module`](https://www.postgresql.org/docs/current/contrib-spi.html) in the PostgreSQL Docs.
+1. Update some rows. This should update both `username`  and `moddate` accordingly.
+
+    ```sql
+    UPDATE spi_test SET content = 'desc1_updated' WHERE id = 1;
+    UPDATE spi_test SET content = 'desc3_updated' WHERE id = 3;
+
+    SELECT * FROM spi_test ORDER BY id;
+    ```
+
+    ```
+    id |    content    | username |          moddate
+    ----+---------------+----------+----------------------------
+      1 | desc1_updated | yugabyte | 2019-09-13 16:56:27.623513
+      2 | desc2         | postgres | 2019-09-13 16:55:53.983306
+      3 | desc3_updated | yugabyte | 2019-09-13 16:56:27.634099
+      4 | desc4         | yugabyte | 2019-09-13 16:55:53.991315
+    (4 rows)
+    ```
 
 ### tablefunc
 
 The `tablefunc` extension provides several table functions. For example, `normal_rand()` creates values, picked using a pseudorandom generator, from an ideal normal distribution. You specify how many values you want, and the mean and standard deviation of the ideal distribution. You use it in the same way that you use `generate_series()`
 
-#### Example
+#### tablefunc example
 
-```plpgsql
+```sql
 CREATE EXTENSION tablefunc;
 
-CREATE TABLE T(k int primary key, v double precision);
+CREATE TABLE t(k int primary key, v double precision);
 
 PREPARE insert_k_v_pairs(int) AS
 INSERT INTO t(k, v)
 SELECT
   generate_series(1, $1),
   normal_rand($1, 1000.0, 10.0);
 ```
-Test it like this
-```plpgsql
+
+Test it as follows:
+
+```sql
 DELETE FROM t;
+
 EXECUTE insert_k_v_pairs(10);
+
 SELECT k, to_char(v, '9999.99') AS v
 FROM t
 ORDER BY k;
 ```
-You'll see a result like this:
+
+You'll see results similar to the following:
```

Comment:
Should we adopt a more formal style? (because it looks professional and recommended by MS and Oracle style guides: e.g. you will)

Reply:
MS has recommended informal writing for a number of years now. I'm going to drag y'all kicking and screaming into a friendlier style. 😁